README.md

```diff
@@ -96,6 +96,14 @@ services:
 >⚠️ Make sure you replace the host, username, and password with values matching your setup, and that the MQTT client is not also used by another application.  
 If the broker is configured to allow anonymous access, the username and password can be omitted.
 
+## Running **cs2mqtt** as Home Assistant Add-On
+
+[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Flupusbytes%2Fcs2mqtt)
```

Comment:
> Do you know if it's possible to use an "add-on repo" from a branch other than main, like if i want to test stuff, but not pollute my main git history?

Sure, that just requires appending `#branch` to the repo URL. So the only change needed for this PR would be here:

```suggestion
[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Flupusbytes%2Fcs2mqtt%23branch-name)
```

Since it's URL encoded here, `%23` instead of `#`.